the-collab-lab/tcl-73-smart-shopping-list
# Final styling changes

## Description

This PR included our final styling changes. We ran through the whole application, adjusting and making final changes to styling after many PR's were all merged.

## Related Issue

Closes #14 

## Acceptance Criteria

- [x] Position of logos on home page
- [x] Position of list owner on list page and manage list page(also background/text color)
- [x] Position of toggle switch (left of/next to sign in/out button?)
- [x] Colors of item purchase priority in dark mode (maybe also light mode, just a bit more intense maybe?)
- [x] Position of delete list button on home page(easy fix with flexbox)
- [x] Styling of lists on home page(border lines need to go back between them)
- [x] Add toasts to the delete functionality from Iryna’s recent issue
- [x] Check item search input clear btn has radix themes applied
- [x] Styled toast messages to light and dark theme
- [x] Added spacing on radio buttons

## Type of Changes

styling, enhancement, accessibility 

## Updates

### Before

![Screenshot 2024-04-05 at 7 00 11 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/135680082/e392427f-0b8b-41eb-9bfb-90858bd455cb)


### After

![Screenshot 2024-04-05 at 6 59 37 PM](https://github.com/the-collab-lab/tcl-73-smart-shopping-list/assets/135680082/6cf71e7f-bbe4-42c0-8350-bba25e8b0eaa)


## Testing Steps / QA Criteria

- `git pull`
- `git checkout pc-final-styling`
- `npm ci`
- `npm run start`
- On home page, note the position of the Github and Collab Lab logos are visible above the navbar in desktop and mobile and their colors change with the theme
- Position of list owner on list page and manage list page are visible above the navbar in desktop and mobile and their colors change with the theme, and stick the the bottom of the screen, above the navbar
- Position of the light to dark toggle switch and sign in/sign out button are positioned in industry standard locations, best suited for mobile use and easy to locate and use on desktop
- On the List page, the text color for the purchase priority badge changes to white text in dark mode for better accessibility
- Position of delete list button on Home page is consistent with the position of the delete button on the List page
- The bottom borders for Lists and Items on the Home page and the List page were added and the colors were made to match the navbar color as a pleasing accent color in light and dark mode
- Toast messages were added to the delete functionality for the List (recent PR)
- Applied accent colors for dark and light theme with radix architecture to item search input clear button
- Note that toast messages are styled to light and dark theme
- Spacing added to Radio buttons for better ease of use on mobile
